docs/user-guide/styling-your-docs.md

```diff
@@ -30,12 +30,11 @@ the [custom themes](#custom-themes) section below.
 
 ![ReadTheDocs](https://docs.readthedocs.org/en/latest/_images/screen_mobile.png)
 
-## Bootstrap and Bootswatch Themes
+## Bootstrap and Bootswatch themes
```

Comment:
@waylan There is some inconsistency in the documentation about whether to use title case or sentence case. I saw the page heading "Styling your docs" and the subheading above "Built-in themes" were sentence case so went with that. FWIW, I do happen to prefer sentence case but will bow to whatever you want to use... Not many people have an editorial style guide for this kind of thing in their repo :-)